packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -176,6 +198,13 @@ export abstract class LoadBalancedServiceBase extends cdk.Construct {
     new cdk.CfnOutput(this, 'LoadBalancerDNS', { value: this.loadBalancer.loadBalancerDnsName });
   }
 
+  protected getCreateDefaultCluster(scope: cdk.Construct, vpc?: ec2.IVpc): ecs.Cluster {
```

Comment:
Can we rename this to getDefaultCluster? Also, shouldn't vpc be required if you're calling this method, you're expecting cluster to be undefined. 

`new ecs.Cluster(stack, DEFAULT_CLUSTER_ID, { vpc })` would pass vpc as undefined if both cluster and vpc were undefined.

Reply:
This is ok, as the change to the cluster will make the vpc optional and create a new VPC.

Reply:
Ah I see, sounds good!